text/0000-infered-types.md

```diff
@@ -124,7 +105,7 @@ do_something(convert(_::new()))
 
 In the thread [[IDEA] Implied enum types](https://internals.rust-lang.org/t/idea-implied-enum-types/18349), many people had a few concerns about this feature. 
 
-These RFCs could create bugs. An example of this is if a function changes has two enum parameters that share common variant names. Because it’s implied, it would still compile with this bug causing UB.
+These RFCs could create bugs. An example of this is if a function changes has two enum parameters that share common variant names. Because it’s implied, it would still compile with this bug createing unintended behavior wharas by specifying the type names, the compiler would thrown an error.
```

Comment:
It is nothing different from `"VariantName".parse().unwrap()` parsing to another type with the same name.

Reply:
It asked me to write drawbacks, I don't know what to say